optimum/commands/export/openvino.py

```diff
@@ -218,6 +249,10 @@ def run(self):
                     "sym": self.args.sym or False,
                     "group_size": -1 if is_int8 else self.args.group_size,
                     "all_layers": None if is_int8 else self.args.all_layers,
+                    "dataset": self.args.dataset,
+                    "num_samples": self.args.num_samples,
+                    "quant_method": "awq" if self.args.awq else None,
```

Comment:
not sure if it should be `QuantizationMethod.AWQ` instead of "awq" or if the configuration takes care of this https://github.com/huggingface/optimum-intel/blob/52875b992ac2248e49063adcf8116f1fb39cacd2/optimum/intel/openvino/quantization.py#L822

Reply:
I hesitated to do it this way because it would require to introduce dependency on `transformers` in this file in order to import `QuantizationMethod`. But now I see that `transformers` is a general requirement of `optimum-intel` so it should be fine I guess.